.github/workflows/ci-nightly.yml

```diff
@@ -0,0 +1,37 @@
+name: CI
```

Comment:
I'd name this CI Nightly

Reply:
I agree! See https://github.com/NCAR/GeoCAT-examples/pull/475